magenta-lib/src/test/scala/magenta/input/DeploymentResolverTest.scala

```diff
@@ -259,27 +249,22 @@ class DeploymentResolverTest extends FlatSpec with ShouldMatchers {
     )
     val deployments = DeploymentResolver.resolve(yaml)
     deployments.size should be (1)
-    deployments.head.isLeft should be(true)
-    val errors = deployments.head.left.get
-    errors should be(List("Template with name nonExistentTemplate (specified in test) does not exist"))
+    deployments.head.left.value should be("test" -> "Template with name nonExistentTemplate does not exist")
   }
 
-  it should "report missing dependencies" in {
```

Comment:
The former is about templates and the latter is about dependencies. Two different concepts and I think it's useful to distinguish between the errors.